minifi/minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -1501,6 +1501,15 @@ boolean getNifiStarted() {
         }
     }
 
+    public void startChangeNotifier() {
```

Comment:
This is just a temporary solution to make this PR work the proper solution will come later in another PR.